OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp

```diff
@@ -919,9 +901,6 @@ formCompleteStorages( const SimTK::State& s, const OpenSim::Storage &aQIn,
     // Need to set column labels before converting rad->deg
```

Comment:
This comment doesn't make sense anymore because the `convertRadiansToDegrees()` lines have been removed.